### PR TITLE
BugFix 1444 - Ignore Dups on Seed insert

### PR DESCRIPTION
**Pull request by shashi165**
### Summary:
While inserting the new seeds, if there are any existing values in the table then it was failing silently without inserting the new seed values.
Changed the insert query to use `INSERT IGNORE` to ignore the dups and continue adding the new values. Also updated the unit tests.

### Unit Tests:
```
1..1378
# tests 1378
# pass  1378

# ok
```
### Coverage Tests
```
=============================== Coverage summary ===============================
Statements   : 98.94% ( 3273/3308 )
Branches     : 97.65% ( 1204/1233 )
Functions    : 99.5% ( 395/397 )
Lines        : 98.94% ( 3254/3289 )
================================================================================
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/central-ledger",
-  "version": "11.2.3",
+  "version": "11.2.4",
```
Comment (elnyry-sam-k) [inline package.json:3]: I think we can consider this new functionality.. 11.3.0?

**Review by vgenev — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (vgenev) [reply package.json:3]: I am ok after that is addressed..

**Review by mdebarros — DISMISSED** [review]
+1

File: package.json (hunk 1 above)
Comment (shashi165) [reply package.json]: Done

**Review by vgenev — APPROVED** [review]
👍

**Review by elnyry-sam-k — DISMISSED** [review]

**Review by vgenev — APPROVED** [review]
go!

**Pull request merged**